packages/website/src/pages/layout/Table.tsx

```diff
@@ -33,12 +31,11 @@ const DemoPage = () => (
                 />
             ),
             emptyState: <TableEmptyStateDemo noPadding layout="vertical" title="Table with empty states" />,
-            consumer: (
-                <TableConsumerDemo
-                    noPadding
-                    layout="vertical"
-                    title="Table with a child component using the hook to re-fetch"
-                />
+            multiSelect: (
+                <TableMultiSelectionDemo noPadding layout="vertical" title="Table with bulk selection of rows" />
+            ),
+            disableRowSelection: (
+                <TableDisableRowSelection noPadding layout="vertical" title="Table with disabled row selection" />
```

Comment:
Is a state disabled row planned? With CSS that directly changes the color of the font?
Or could a simple example be added?

Reply:
I will add a prop like the getRowId function called getRowAttributes. It will spread the attributes on the table row (`<tr>`). This way you will be able to do something like

```tsx
getRowAttributes={(rowData) => ({'data-disabled': !rowData.enabled})}
```

and then target that attribute with css

Reply:
I will do that in a separate PR